src/cwl_platform/sevenbridges_platform.py

```diff
@@ -194,6 +194,32 @@ def _list_files_in_folder(self, project=None, folder=None, recursive=False):
             file_list = self.api.files.get(id=parent.id).list_files().all()
         return file_list
 
+    def clean_old_output_file(self, project, file_name):
+        ''' 
+        Find and rename output file from previous runs.
+
+        :param project: The project to clean old output file
+        :param file_name: The filename that needs to be renamed
+        '''
+        file_to_rename = self.api.files.query(project=project, names=[file_name])
+        if len(file_to_rename) == 0: # Do nothing if file not exists
+            return True
+
+        file_to_rename = file_to_rename[0]
+        file_list = self.api.files.query(project=project).all()
+        existing_filenames=[]
+        for x in file_list:
+            if file_name in x.name:
+                existing_filenames += [x.name]
+        i=1
+        new_filename = "_" + str(i) + "_" + file_name
+        while new_filename in existing_filenames:
+            i+=1
+            new_filename = "_" + str(i) + "_" + file_name
+
```

Comment:
Instead of explicitly renaming the file here, can you reuse/call the rename_file method?